python/semantic_kernel/core_skills/connversion_summary_skill.py

```diff
@@ -0,0 +1,56 @@
+from semantic_kernel.orchestration.sk_context import SKContext
+from semantic_kernel.skill_definition import sk_function
+from semantic_kernel.kernel import Kernel
+from semantic_kernel.text import text_chunker
+
+from python.semantic_kernel.text.function_extension import aggregate_chunked_results_async
```

Comment:
I think this should be 

`semantic_kernel.text.function_extension`

Reply:
fixed.